docs/admin/install/docker.rst

```diff
@@ -409,11 +409,22 @@ Machine translation settings
 
     Enables :ref:`mymemory` machine translation and sets
     :setting:`MT_MYMEMORY_EMAIL` to :envvar:`WEBLATE_ADMIN_EMAIL`.
+    
+   **Example:**
+
+    .. code-block:: yaml
```

Comment:
Empty line missing between code-block and actual example, this fails to build: https://cloud.drone.io/WeblateOrg/weblate/444/2/2

Reply:
```suggestion
    .. code-block:: yaml
    
```